modules/gapi/include/opencv2/gapi/gtype_traits.hpp

```diff
@@ -26,7 +26,7 @@ namespace detail
     // a double dispatch
     enum class ArgKind: int
     {
-        OPAQUE,       // Unknown, generic, opaque-to-GAPI data type - STATIC
+        GOPAQUE,      // Unknown, generic, opaque-to-GAPI data type - STATIC
```

Comment:
Add compatibility entry:
```
#if !defined(OPAQUE) && !defined(CV_DOXYGEN)
    OPAQUE = GOPAQUE,  // deprecated
#endif
```

Reply:
thanks! by the way, why CV_DOXYGEN is required?

Reply:
ah, I think I got it. this way `OPAQUE` won't be seen in documentation

Reply:
updated